.vscode/launch.json

```diff
@@ -8,13 +8,24 @@
       "type": "node",
       "request": "launch",
       "name": "Debug Server",
-      "preLaunchTask": "ts-build",
```

Comment:
We need to make a note that this step in the build causes windows to fail on the debug start up. It might even be worth removing this line and just forcing every one to run this manually.

Reply:
I have commented this out in the configuration, and clarified that `npm run watch-ts` needs to be run manually.